style.css

```diff
@@ -0,0 +1,721 @@
+:root {
+
+  --primary-color: rgba(248, 247, 255, 1);
+  --main-font: 'Bakbak One','Poppins', Helvetica, Arial, sans-serif;
```

Comment:
Nice work! You also can add secondary color and secondary font if you want!

And let a little fix it. We need only one main font here, one font alternative and one family.

Poppins font looks like secondary font for project. What do you think?